linux_os/guide/system/software/updating/ensure_gpgcheck_local_packages/oval/fedora.xml

```diff
@@ -0,0 +1,25 @@
+<def-group>
+  <definition class="compliance" id="ensure_gpgcheck_local_packages" version="1">
+    <metadata>
+      <title>Ensure gpgcheck Enabled for Local Packages</title>
+      <affected family="unix">
+        <platform>multi_platform_fedora</platform>
+      </affected>
+      <description>The localpkg_gpgcheck option should be used to ensure that checking 
+      of an RPM package's signature always occurs prior to its
+      installation.</description>
+    </metadata>
+    <criteria>
+      <criterion comment="check value of localpkg_gpgcheck in /etc/dnf/dnf.conf" test_ref="test_yum_ensure_gpgcheck_local_packages" />
+    </criteria>
+  </definition>
+  <ind:textfilecontent54_test check="all" check_existence="all_exist" comment="check value of localpkg_gpgcheck in /etc/dnf/dnf.conf" id="test_yum_ensure_gpgcheck_local_packages" version="1">
+    <ind:object object_ref="object_yum_ensure_gpgcheck_local_packages" />
+  </ind:textfilecontent54_test>
+  <ind:textfilecontent54_object id="object_yum_ensure_gpgcheck_local_packages" comment="localpkg_gpgcheck set in /etc/dnf/dnf.conf" version="1">
+    <ind:filepath>/etc/dnf/dnf.conf</ind:filepath>
```

Comment:
Maybe this OVAL can be merged as well?

Reply:
The OVAL can be much simpler, with jinja the OVAL can be much simpler and "customized" for each product.

Reply:
Yes, using Jinja macros seems as the best way to handle those differences.